classes/ValidFormBuilder/FieldValidator.php

```diff
@@ -405,7 +405,7 @@ public function validate($intDynamicPosition = 0)
                 }
             }
         } elseif (strlen($value) == 0) {
-            if ($this->__required && $intDynamicPosition == 0) {
+            if (($this->__required && $intDynamicPosition == 0) || !!$this->__field->getMeta('dynamicRemoveLabel', false)) {
```

Comment:
What does `!!$this->__field->getMeta('dynamicRemoveLabel', false)` actual mean? Please try to use descriptive coding so others know what's going on. More code doesn't make it slower ;-)